api/services/passport.js

```diff
@@ -60,7 +60,12 @@ passport.callback = (req, res) => {
         }
       });
     } else {
-      res.redirect('/?login_failed=1');
```

Comment:
don't need this query parameter-based messaging any more with the new message flash stuff